presentation/home/src/main/java/com/dhc/home/main/HomeContract.kt

```diff
@@ -21,10 +20,20 @@ class HomeContract {
         val selectedMissionInfo: SelectChangeMission = SelectChangeMission(),
         val homeInfo: HomeUiModel = HomeUiModel(),
         val finishTodayMission: Boolean = false,
+        val todaySavedMoney: String = "",
     ): UiState {
+        val remainingMissionCount: Int
+            get() = 4 - getFinishedMissionCount()
```

Comment:
@ddyeon 
```suggestion
            get() = homeInfo.todayDailyMissionList.size - getFinishedMissionCount()
```
이거 이렇게 하면 더 안전할듯 ??

Reply:
좋다 get() = getMissionIdList() - getFinishedMissionCount() 이걸로 수정할게!